src/Libraries/CoreNodeModels/Logic/If.cs

```diff
@@ -72,4 +78,78 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
             };
         }
     }
+
+    [NodeName("If")]
+    [NodeCategory(BuiltinNodeCategories.LOGIC)]
+    [NodeDescription("ScopeIfDescription", typeof(Resources))]
+    [OutPortTypes("Function")]
+    [IsDesignScriptCompatible]
+    [AlsoKnownAs("DSCoreNodesUI.Logic.If")]
+    public class RefactoredIf : NodeModel
```

Comment:
Since this is going to be a public class, is "RefactoredIf" name fine with everyone? Are we going to obsolete the old node in 3.0 and then rename this class name to "IF"?

Reply:
Maybe a more descriptive name and summary? Is there any reason not to obsolete the old node right now?

Reply:
Sorry I meant, if we are going to remove this old node completely in 3.0. Currently the old one is obsoleted and it has a summary under BuildOutputAst(). Do you want me to add any other information?